packages/itwinui-react/src/core/Carousel/CarouselSlide.tsx

```diff
@@ -51,7 +51,6 @@ export const CarouselSlide = React.forwardRef((props, ref) => {
     <Box
       as='li'
       className={cx('iui-carousel-slider-item', className)}
-      role='tabpanel'
```

Comment:
i believe you need `role='group'` when using `aria-roledescription`

Reply:
that's interesting, because [APG](https://www.w3.org/WAI/ARIA/apg/patterns/carousel/) suggests using `role=group` (check the heading "Basic carousel elements")

i believe it might be because we are using `li`. we could make it a generic `div`. if we do that, the parent `ol` would also need to be a `div`.

Reply:
Done